book/logging.md

```diff
@@ -114,70 +96,14 @@ allows organizations to collect, index, search, analyze, and visualize machine-g
-| OpenShift infrastructure | rh_rhtap_production_audit | [production-rhtap-audit-logs](https://rover.redhat.com/groups/group/production-rhtap-audit-logs)             |
-| OpenShift audit          | rh_rhtap_production_audit | [production-rhtap-audit-logs](https://rover.redhat.com/groups/group/production-rhtap-audit-logs)             |
-| AWS VPC Flow Logs        | rh_rhtap_production_audit | [production-rhtap-vpc-flow-logs](https://rover.redhat.com/groups/group/production-rhtap-vpc-flow-logs)       |
-
-
-#### OpenShift Logging Configuration
-https://github.com/redhat-appstudio/infra-deployments/tree/main/components/monitoring/logging
-
-#### Secrets
-Secrets are stored in [Hashicorp Vault](https://vault.devshift.net).
-
-#### RHTAP Staging Environment
-| **Secret Name**                            | **Vault Path**            |
-|:-------------------------------------------|:--------------------------|
-| splunk-forwarder-rhtap-staging-application | [stonesoup/staging/monitoring/logging/fluentd/splunk-forwarder-rhtap-staging-application](https://vault.devshift.net/ui/vault/secrets/stonesoup/show/staging/monitoring/logging/fluentd/splunk-forwarder-rhtap-staging-application) |
-| splunk-forwarder-rhtap-staging-audit       | [stonesoup/staging/monitoring/logging/fluentd/splunk-forwarder-rhtap-staging-audit](https://vault.devshift.net/ui/vault/secrets/stonesoup/show/staging/monitoring/logging/fluentd/splunk-forwarder-rhtap-staging-audit) |
-| rh_rhtap_staging_vpc                       | [stonesoup/staging/monitoring/logging/rh_rhtap_staging_vpc](https://vault.devshift.net/ui/vault/secrets/stonesoup/show/staging/monitoring/logging/rh_rhtap_staging_vpc) |
-
-#### RHTAP Production Environment
-| **Secret Name**                               | **Vault Path**            |
-|-----------------------------------------------|:--------------------------|
-| splunk-forwarder-rhtap-production-application | [stonesoup/production/monitoring/logging/fluentd/splunk-forwarder-rhtap-production-application](https://vault.devshift.net/ui/vault/secrets/stonesoup/show/production/monitoring/logging/fluentd/splunk-forwarder-rhtap-production-application)  |
-| splunk-forwarder-rhtap-production-audit       | [stonesoup/production/monitoring/logging/fluentd/splunk-forwarder-rhtap-production-audit](https://vault.devshift.net/ui/vault/secrets/stonesoup/show/production/monitoring/logging/fluentd/splunk-forwarder-rhtap-production-audit) |
-| rh_rhtap_staging_vpc                          | [stonesoup/production/monitoring/logging/rh_rhtap_production_vpc](https://vault.devshift.net/ui/vault/secrets/stonesoup/show/production/monitoring/logging/rh_rhtap_production_vpc) |
-
-**Notice** Secrets are also backed-up in the o11y team's BitWarden vault.
-
-### Logs retention
-Logs are being retained for 60 days.
+### CloudWatch
```

Comment:
I don't think we should say much about cloudwatch, because we're (hopefully) going to shut it down.